docs/source/configuration/core-addons.md

```diff
@@ -0,0 +1,24 @@
+# Volto Core add-ons
```

Comment:
This filename should align with its title. Please rename and move to `developer-guidelines/core-add-ons.md`, and update the indexes accordingly.

Reply:
Done.